cmd/kops/create_cluster.go

```diff
@@ -472,6 +481,27 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 
 	if c.Cloud != "" {
 		cluster.Spec.CloudProvider = c.Cloud
+
+		if c.Cloud == "vsphere" {
+			if cluster.Spec.CloudConfig == nil {
+				cluster.Spec.CloudConfig = &api.CloudConfiguration{}
+			}
+
+			if c.VSphereServer == "" {
+				return fmt.Errorf("vsphere-server is required for vSphere. Set vCenter URL Ex: 10.192.10.30 or myvcenter.io (without https://)")
+			}
+			cluster.Spec.CloudConfig.VSphereServer = c.VSphereServer
```

Comment:
Why should a k8s cluster spec have vSphere server details? Since kops will be used on a particular VC, storing this info here seems redundant. Can you explain what is the purpose this is serving?